x-pack/plugins/secops/public/components/range_date_picker/quick_select_popover/commonly_used.tsx

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiFlexGrid, EuiFlexItem, EuiLink, EuiSpacer, EuiText, EuiTitle } from '@elastic/eui';
+import { getOr } from 'lodash/fp';
+import moment, { Moment } from 'moment';
+import React from 'react';
+import { pure } from 'recompose';
+import { DateType, RecentlyUsedI } from '..';
+
+export enum DatePickerOptions {
+  today = 'today',
+  yesterday = 'yesterday',
+  thisWeek = 'this-week',
+  weekToDate = 'week-to-date',
+  thisMonth = 'this-month',
+  monthToDate = 'month-to-date',
+  thisYear = 'this-year',
+  yearToDate = 'year-to-date',
+}
+
+const commonDates: Array<{ id: string; label: string }> = [
```

Comment:
It would be better types IMHO, if you do this:

```ts
const commonDates: Array<{ id: DatePickerOptions; label: string }> = [
```
then on line 71 you can remove your `as` cast.